client/transport/jsonrest/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.github.a2asdk</groupId>
+        <artifactId>a2a-java-sdk-parent</artifactId>
+        <version>0.3.0.Beta1-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+    <artifactId>a2a-java-sdk-client-transport-jsonrest</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Java SDK A2A Client Transport: JSON+HTTP/REST</name>
+    <description>Java SDK for the Agent2Agent Protocol (A2A) - JSON+HTTP/REST Client Transport</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>a2a-java-sdk-client-config</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>a2a-java-sdk-common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>a2a-java-sdk-spec</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>a2a-java-sdk-spec-grpc</artifactId>
```

Comment:
@ehsavoie @kabir Just a general question. Was just taking a look at the dependency tree for `a2a-java-sdk-client-transport-jsonrest` and the `a2a-java-sdk-spec-grpc` dependency is also bringing in dependencies on ` io.grpc:grpc-protobuf` and `io.grpc:grpc-stub` in addition to the `protobuf` related dependencies.

I think only the `protobuf` ones like ` com.google.protobuf:protobuf-java-util` and ` com.google.protobuf:protobuf-java` are needed for the rest transport, right?

Just thinking it might be good to avoid brining in the `io.grpc` ones.

WDYT?

Reply:
For the server side, I noticed the same for `a2a-java-sdk-transport-jsonrest`.

Reply:
Should be fixed